modules/bloxone_anycast/outputs.tf

```
@@ -0,0 +1,19 @@
+output "anycast_config" {
+  description = "The anycast config"
+  value       = bloxone_anycast_config.ac
+}
+
+output "anycast_hosts" {
+  description = "Map of anycast hosts"
+  value       = { for k, v in bloxone_anycast_host.this : k => v }
```

```suggestion
  value       = bloxone_anycast_host.this
```

The for function is unnecessary here.

---

not needed